p2p_wallet/Scenes/Main/BankTransfer/StrigaRegistration/FirstStep/StrigaRegistrationFirstStepViewModel.swift

```diff
@@ -153,9 +153,18 @@ private extension StrigaRegistrationFirstStepViewModel {
 
     func fetchPhoneNumber(data: StrigaUserDetailsResponse) async {
         let countries = try? await countriesService.fetchCountries()
-        let metadata = await self.strigaMetadata.getStrigaMetadata()
+        let metadataService: WalletMetadataService = Resolver.resolve()
```

Comment:
why have you changed the source for metadata from strigaMetadata.getStrigaMetadata to WalletMetadataService ?

Reply:
I've only set priority from Local state to Striga metadata to web3 auth